x-pack/plugins/observability_solution/dataset_quality/public/components/quality_indicator/percentage_indicator.tsx

```diff
@@ -5,20 +5,63 @@
  * 2.0.
  */
 
-import { EuiText } from '@elastic/eui';
+import { EuiIcon, EuiText, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
 import { FormattedNumber } from '@kbn/i18n-react';
 import React from 'react';
 import { mapPercentageToQuality } from './helpers';
 import { QualityIndicator } from './indicator';
 
-export function QualityPercentageIndicator({ percentage = 0 }: { percentage?: number }) {
+const FEW_DEGRADED_DOCS_THRESHOLD = 0.0005;
+
+export function QualityPercentageIndicator({
+  percentage,
+  degradedDocsCount,
+}: {
+  percentage: number;
+  degradedDocsCount?: number;
+}) {
   const quality = mapPercentageToQuality(percentage);
 
-  const description = (
-    <EuiText size="s">
-      <FormattedNumber value={percentage} />%
-    </EuiText>
+  const isFewDegradedDocsAvailable = percentage && percentage < FEW_DEGRADED_DOCS_THRESHOLD;
+
+  const description = isFewDegradedDocsAvailable ? (
+    <DatasetWithFewDegradedDocs percentage={percentage} degradedDocsCount={degradedDocsCount} />
+  ) : (
+    <DatasetWithManyDegradedDocs percentage={percentage} />
   );
 
   return <QualityIndicator quality={quality} description={description} />;
 }
+
+const DatasetWithFewDegradedDocs = ({
+  percentage,
+  degradedDocsCount,
+}: {
+  percentage: number;
+  degradedDocsCount?: number;
+}) => {
+  return (
+    <EuiText size="s">
+      ~<FormattedNumber value={percentage} />%{' '}
```

Comment:
[nit] Shouldn't this just be `~0%`? We don't need to use the Formatted number if we know we are in this case already